.github/workflows/ci.yml

```diff
@@ -106,6 +106,29 @@ jobs:
           git commit --reset-author --amend -m "Update from GitHub Actions"
           git push --force origin gh-pages
 
+  copyright_check:
+    runs-on: [ubuntu-20.04]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run check
+        run: |
+          for changed_file in $(git diff --diff-filter=d --name-only ${{ github.event.pull_request.base.sha }}); do
+            echo "Handle ${changed_file}"
+            if [[ -z $(grep "Copyright (c).*-.* Intel Corporation" ${changed_file}) ]]; then
+              sed -i "/Copyright (c).* Intel Corporation/s/[0-9]\{4\} /& -$(date +%Y) /" ${changed_file}
+            else
+              sed -i "/Copyright (c).*-.* Intel Corporation/s/[0-9]\{4\} /$(date +%Y) /g" ${changed_file}
+            fi
+          done
```

Comment:
```suggestion
          sed -i \
              -e "/Copyright (c).*-.* Intel Corporation/s/[0-9]\+ /$(date +%Y) /" \
              -e "/Copyright (c) [0-9]\+ Intel Corporation/s/[0-9]\+/&-$(date +%Y)/" \
              $(git diff --diff-filter=d --name-only ${{ github.event.pull_request.base.sha }})
```

Reply:
Good idea, thanks

Reply:
We have problem in case with correct year.
See here https://github.com/oneapi-src/oneTBB/runs/5322735202?check_suite_focus=true